### ISSUE TITLE
Could you please give a full usable example of cross_file.txt?

### ISSUE THREAD

**Issue by zeerd** [mesonbuild/meson#7036]

**Comment by eli-schwartz** [comment]
Is there a problem with https://github.com/mesonbuild/meson/tree/master/cross ?

**Comment by zeerd** [comment]
Ah! Thank you!

But as a sugget, I think you'd better add this url into the https://mesonbuild.com/Cross-compilation.html. So that everyone could find it.

**Comment by michaelbadcrumble** [comment]
Agreed not only that but this link would be perfect for cross compilation tips in [Meson build tips](https://github.com/michaelbadcrumble/mesonbuild_tips)

**Comment by bonzini** [comment]
Fixed by https://mesonbuild.com/Cross-compilation.html.
